api/script/seed.js

```diff
@@ -1,27 +1,28 @@
-const path = require("path");
```

Comment:
probably doesn't need to happen here but I think we should switch away from including a data.zip and go to actual knexfile seeding so that users (or us) can properly seed and demo the app on all of our supported databases, not just sqlite

Reply:
Yeah for now the simplest way is to keep sqlite3 with this way of seeding to be honest.